### PR TITLE
feat: delete account confirmation flow

### PR DESCRIPTION
**Pull request by shirleyodeng**
## Summary

This PR changes the delete account flow with a confirmation flow.

**Settings tab**

Screenshot 2022-08-22 at 09 14 56

**Email**

Screenshot 2022-08-22 at 19 06 54

## Notion link

https://talentprotocol.notion.site/Change-delete-account-flow-e8e0adc39e394bcd92283cbd553d5379

## How to test?

- Go to "Settings"
- Scroll down to "Close Account" and click "Send delete account confirmation email"
- Open confirmation email
- Click "Close my Talent Protocol account" CTA
- Account should be deleted and redirected to the home page (login page)

### CODE REVIEW

**Comment by shirleyodeng** [comment]
@RubenSousaDinis - set the expiry of the delete account token to be an hour - thinking if this should be added to the email to let the user know?

**Review by RubenSousaDinis — APPROVED** [review]
Great work!

File: app/controllers/api/v1/users/delete_account_tokens_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class API::V1::Users::DeleteAccountTokensController < ApplicationController
+  def create
```
Comment (RubenSousaDinis) [inline app/controllers/api/v1/users/delete_account_tokens_controller.rb:2]: Can we have a request spec for this action? 🥺

**Review by RubenSousaDinis — COMMENTED** [review]
Just one small request.

File: app/controllers/api/v1/users/delete_account_tokens_controller.rb (hunk 1 above)
Comment (shirleyodeng) [reply app/controllers/api/v1/users/delete_account_tokens_controller.rb:2]: @RubenSousaDinis of course 😉 - just noticed there wasn't any other controller specs 🙈

Comment (RubenSousaDinis) [reply app/controllers/api/v1/users/delete_account_tokens_controller.rb:2]: Eheh yes, we're only doing request specs

Comment (shirleyodeng) [reply app/controllers/api/v1/users/delete_account_tokens_controller.rb:2]: Ahhh right, spotted them. Added one :) > https://github.com/talentprotocol/web-dapp/pull/660/commits/8f825db386d290cf7a57ca77f29e3086a27fa0ba

**Pull request merged**